code/stable/glassbr/SRS/HTML/GlassBR_SRS.html

```diff
@@ -388,9 +388,7 @@ <h2>Table of Symbols</h2>
                 </tr>
                 <tr>
                   <td><em>t</em></td>
-                  <td>
-                    Nominal thickness <em>t&thinsp;&isin;&thinsp;{2.5,2.7,3.0,4.0,5.0,6.0,8.0,10.0,12.0,16.0,19.0,22.0}</em>
-                  </td>
+                  <td>Nominal thickness</td>
```

Comment:
Hmm, why did we lose some information from your changes? Did it just move?

Reply:
I don't think it's something we're 'losing' necessarily, but moving to where it should be (or at least closer to where it should be, but at least in the form it should be in) -- a constraint. The reason why it was in the description of nominal thickness to begin with was because we previously removed 'enumerated reals/integers' constraints.